patches/server/0372-implement-optional-per-player-mob-spawns.patch

```diff
@@ -694,10 +694,13 @@ index 95e5660e6cb1afb5ebdb3dbbe59a07c879bddb4b..88145f04989c71a686aae1b486087ecd
 +                }
 +                difference = (minDiff == Integer.MAX_VALUE) ? 0 : minDiff;
 +            }
++            if (difference <= 0) {
```

Comment:
remove this and keep the `difference > 0` condition below

Reply:
got it!
Should I rebase commit?

Reply:
We can handle rebasing/squashing on merge, thanks though